corehq/apps/callcenter/fixturegenerators.py

```diff
@@ -45,26 +45,26 @@ def should_sync(domain, last_sync, utcnow=None):
 class IndicatorsFixturesProvider(object):
     id = 'indicators'
 
-    def __call__(self, user, version, last_sync=None, app=None):
-        assert isinstance(user, CommCareUser)
+    def __call__(self, restore_user, version, last_sync=None, app=None):
+        assert isinstance(restore_user, OTARestoreUser)
```

Comment:
Guess I just can't see where the `restore_user` is being passed in instead of the `CommCareUser`